### DISCUSSION TITLE
Tolerance for cartesian waypoint

### DISCUSSION THREAD

**Discussion by shruti-gangopadhyay** [tesseract-robotics/tesseract_ros#169] [Q&A]
Hello, I want to add a tolerance to the orientation while passing a cartesian waypoint. Right now I see that tolerance for cartesian waypoint has not been implemented. Is there some way n the current implementation by which the tolerance may be added?

**Comment by Levi-Armstrong** [comment]
We held off implementing this mostly because we are planning on migrating to trajopt_ifopt which makes it a lot easier to implement tolerance, but trajopt_ifopt still needs some work before that can happen.

**Reply by shruti-gangopadhyay** [reply]
Oh I see. So currently there is no way to do this then? With the default or with ifopt?
